lib/matplotlib/axes/_axes.py

```diff
@@ -6540,6 +6540,11 @@ def hist(self, x, bins=None, range=None, density=False, weights=None,
         --------
         hist2d : 2D histograms
 
+        Notes
+        -----
+        - For large numbers of bins, 'step' and 'stepfilled' can be
```

Comment:
should the strings be in double backticks to render as code?

Reply:
I will leave this as is then.